src/transformers/models/vit/modeling_flax_vit.py

```diff
@@ -171,6 +179,7 @@ def __call__(self, hidden_states, deterministic: bool = True, output_attentions:
         attn_weights = dot_product_attention_weights(
             query_states,
             key_states,
+            mask=attention_mask,
```

Comment:
same comment as above, no `mask` argument. Should be expanded to a tensor and pass as bias. Here is how it's done in bert

https://github.com/huggingface/transformers/blob/6c08840628a22a4d53ae563d1041479649d1a8e7/src/transformers/models/bert/modeling_flax_bert.py#L229-L254